.github/workflows/ci.yml

```diff
@@ -10,7 +10,7 @@ permissions:
 
 jobs:
   ci:
-    uses: input-output-hk/catalyst-ci/.github/workflows/ci.yml@master
+    uses: input-output-hk/catalyst-ci/.github/workflows/ci.yml@tests/typhon-registration
```

Comment:
You should not change this because it refers to the master branch of this repo input-output-hk/catalyst-ci/.github/workflows/ci.yml